M1L3HW/M1L3HW/Program.cs

```diff
@@ -4,9 +4,122 @@ namespace M1L3HW
 {
     internal class Program
     {
+        static string alphabet = "abcdefghijklmnopqrstuvwxyz";
+        static string lettersToUpper = "aeidhj";
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            Console.WriteLine("Add an array length");
+
+            // Catch an exception while user inputs not a number or number that < 0.
+            int arrLength = -1;
```

Comment:
The code can be separated into the standalone methods (functions), so it would be much easier to read and understand + your beautiful comments would make more sense

Reply:
U can make a separation based on the task parts